README

```diff
@@ -220,6 +225,27 @@
 
         IMPORTANT: Complete all Guest VM Install prerequisites listed in the Prerequisites
                    section above BEFORE running this script with the "--install" option.
+                   
+      
+      --input-file INPUTFILE
+        Use this option to provide your own custom input file to be used for host tests. 
+        Make sure the input file has the same structure and syntax as the config/tests/host/sanity.cfg 
+        file already in this repository.  This option is helpful if you wish to quickly reference 
+        your own host tests config file without editing the default config/tests/host/sanity.cfg file.
+      
```

Comment:
@rquander I would like to get inputs from @narasimhan-v on this

Reply:
@rquander There is a confusion between input file and config file here. I can provide a sample input file and also update this and other relevant sections of README in a separate commit.